CHANGELOG.md

```diff
@@ -1,67 +1,109 @@
+<!-- markdownlint-disable MD024 -->
+
 # Change Log
+
 All notable changes to the "azurefunctions" extension will be documented in this file.
 
+## 0.8.0 - 2018-04-05
+
+### Added
+
+- Upload and download application settings
+- Prompt to automatically install the Azure Functions Core Tools when possible
```

Comment:
Should we be more specific here?  I'm worried users won't be able to anticipate when this will happen since it's not specified that it occurs after a new project is created.

Reply:
I'll let you handle that as a part of your PR if you want. I honestly wasn't sure what the final feature would look like haha